syntax = "proto3";

package flwr.proto;

// MinIO messages
message MessageMinIO {
  string bucket_name = 1;
  string source_file = 2;
}
